core/src/main/resources/error/error-classes.json

```diff
@@ -1191,6 +1191,12 @@
     ],
     "sqlState" : "42805"
   },
+  "OUTPUT_PATH_ALREADY_EXISTS" : {
+    "message" : [
+      "Path <outputPath> already exists."
```

Comment:
Can we suggest something to users, like setting the save mode to a value  `Overwrite = true`?